data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -211,13 +211,17 @@ then
 	# 1. read the controller config rendered by MachineConfigOperator
 	# 2. read the default MachineConfigPools rendered by MachineConfigOperator
 	# 3. read any additional MachineConfigs that are needed for the default MachineConfigPools.
-	mkdir --parents /etc/mcc/bootstrap /etc/mcs/bootstrap /etc/kubernetes/manifests
+	mkdir --parents /etc/mcc/bootstrap /etc/mcs/bootstrap /etc/kubernetes/manifests /etc/kubernetes/static-pod-resources
 	cp mco-bootstrap/bootstrap/manifests/* /etc/mcc/bootstrap/
 	cp openshift/* /etc/mcc/bootstrap/
 	# 4. read ImageContentSourcePolicy objects generated by the installer
 	cp manifests/* /etc/mcc/bootstrap/
 	cp auth/kubeconfig-kubelet /etc/mcs/kubeconfig
 	cp mco-bootstrap/bootstrap/machineconfigoperator-bootstrap-pod.yaml /etc/kubernetes/manifests/
+        if [ -d mco-bootstrap/baremetal/manifests ]; then
+            cp mco-bootstrap/baremetal/manifests/* /etc/kubernetes/manifests/
+            cp -r mco-bootstrap/baremetal/static-pod-resources/* /etc/kubernetes/static-pod-resources/
```

Comment:
nit: i think this script uses full names for flags `--recursive`

Reply:
> https://github.com/openshift/installer/pull/2119#issuecomment-516652885

@stbenjam will you be filing follow-ups to fix this..

Reply:
The other well known shell commands (`sed -i -e`, `rm -rf`) do not use long flags in this script. If you really want it this way, this  fixes them all: https://github.com/openshift/installer/pull/2130